modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc

```diff
@@ -38,6 +38,17 @@ The contents of a concept module give the user descriptions and explanations nee
 * Avoid including action items. Action items belong in procedure modules. However, in some cases a concept or reference can include suggested actions when those actions are simple, are highly dependent on the context of the module, and have no place in any procedure module. In such cases, ensure that the heading of the concept or reference remains a noun phrase and not a gerund.
 
 
+////
+Include titles and alternative text descriptions for images.
+Alternative text should provide a textual, complete description of the image as a full sentence.
+Images should never be the sole means of conveying information but only to supplement the text.
+Take careful consideration before adding screenshots or other images that might quickly go out of date and create a maintenance burden on documentation.
+Ensure you request assistance from the Red Hat illustration team for any diagrams.
+Use the CCS diagram request form (ask in chat if you don't know where to find it).
+////
+//.Image title
+//image::image-file.png[This illustration depicts a deployment scenario where something does something and has an effect.]
+
```

Comment:
@oraNod Thanks. My suggestion below, to simplify a tad, and also to avoid referring to CCS and Red Hat requirements. We've been trying to keep the mod docs project as truly open-source-friendly as possible, specifically trying to avoid requirements imposed by RH, CCS, Pantheon, etc. 

```
////
Include titles and alternative text descriptions for images.
Alternative text should provide a textual, complete description of the image as a full sentence.
Images should never be the sole means of conveying information and should only supplement the text.
Avoid screenshots or other images that might quickly go out of date and that create a maintenance burden on documentation.
////
//.Image title
//image::image-file.png[This illustration depicts a deployment scenario where something does something and has an effect.]
```